python/process_cmd_params.py

```diff
@@ -0,0 +1,80 @@
+# Process the command line arguments as user_email, user_param and user_selection, exception handling also happens here
+
+import sys
+import csv
+
+def process():
+    # read the labels file, to be used to convert user input labels to strings that other part of the application can recognize
+    label_dict={}    
+    with open("./data/labels.csv") as f:
+        csv_reader=csv.reader(f, delimiter=',')
+        for row in csv_reader:
+            label_dict[row[1]]=row[0]
+    # process command line arguments        
+    user_email=sys.argv[1]
```

Comment:
I'd recommend refactoring this to use the [argparse](https://docs.python.org/3/library/argparse.html) module. It lets you store clargs as named variables, which would make this function much more readable.